swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -615,6 +625,14 @@ let default_translations = {
       share: {
         share: "Jaa",
         share_id: "Jaa tunnus",
+        share_id_tooltip:
+          "When another project wants share a folder " +
```

Comment:
Initial translation for this:
Mikäli toisesta projektista halutaan jakaa kansio projektisi kanssa: Valitse tämä nappi ja lähetä Jakotunnus toisen projektin jäsenelle. Halutessasi jakaa kansion toisen projektin kanssa, pyydä jakotunnusta joltakin kyseisen projektin jäseneltä.

Reply:
I added the translation. Thanks 👍